tests/test.rs

```diff
@@ -133,6 +133,107 @@ fn debug() {
     println!("{full:?}");
 }
 
+// a dummy defmt logger and timestamp implementation, for testing
+#[cfg(test)]
```

Comment:
Isn't this redundant? The whole file is only compiled for `test`.

Reply:
Yes -- this is cruft from an earlier revision where I was trying to get a single dummy implementation that would work for all tests and doctests, but doctests especially resist conditional compilation like this.